components/service/glean/README.md

```diff
@@ -79,7 +79,7 @@ class SampleApplication : Application() {
         // Call setUploadEnabled first, since Glean.initialize
         // might send pings if there are any metrics queued up
         // from a previous run.
-        Glean.setUploadEnabled(Settings.isTelemetryEnabled)
+        Glean.setUploadEnabled(Settings.isTelemetryEnabled, applicationContext)
```

Comment:
I think it's ok to revert back to the previous version of the API: we store the applicationContext in other singletons. See [here](https://github.com/mozilla-mobile/android-components/blob/01837eee6cb00b698daafe96636f41c45cd88b34/components/service/glean/src/main/java/mozilla/components/service/glean/storages/UuidsStorageEngine.kt#L17-L21).